src/devices/Bmx280/Bme280.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+
+namespace Iot.Device.Bmx280
+{
+    public class Bme280 : BmxBase
+    {
+        private const byte Signature = 0x60;
+        public const byte Address = 0x76;
```

Comment:
I think it should be called **I2cAddress**

Reply:
we should agree on naming. What about DefaultI2cAddress for all those cases? At least it's very explicit.

Reply:
I like @Ellerbach proposal, `DefaultI2cAddress` makes sense to me

Reply:
What about `DefaultI2cAddressPrim`  and `DefaultI2cAddressSec`?

The Bme280 and Bmp280 have two address, dependes of SDO pin conection
[BMP280](https://cdn-shop.adafruit.com/datasheets/BST-BMP280-DS001-11.pdf
) Page 28
[BME280](https://cdn-shop.adafruit.com/datasheets/BST-BME280_DS001-10.pdf) Page 31

SDO to GND    -> 0x76
SDO to VDDIO -> 0x77

Reply:
use enum type
```C#
public enum I2cAddress : byte
{

}
```

Reply:
I'd go with the just Default - if user already looks in the datasheet to set physical pins they can as well check the address

Reply:
> I'd go with the just Default - if user already looks in the datasheet to set physical pins they can as well check the address

Fair. I'll remove the second one. It's anyway a specific setup